docs/breaking-changes.md

```diff
@@ -50,6 +50,25 @@ If your crash ingestion server does not support compressed payloads, you can
 turn off compression by specifying `{ compress: false }` in the crash reporter
 options.
 
+### Deprecated: `remote` module
+
+The `remote` module is deprecated in Electron 12, and will be removed in
```

Comment:
Can we utilize this message to further indicate that you shouldn't just replace and forget.  Let's use it as another chance to educate why using `remote` is bad.

Reply:
The @electron/remote repository has a big warning at the top. I think that's enough.